include/compiler.h

```diff
@@ -28,3 +28,12 @@
 
 #define LIKELY(x) __builtin_expect(!!(x), 1)
 #define UNLIKELY(x) __builtin_expect(!!(x), 0)
+
+#if __clang__
```

Comment:
I don't think we need these now. But this is actually a good place to check if these are present and `ENABLE_ASAN` is false. We should probably put a static_assert in there for that. WDYT?